CHANGELOG.md

```diff
@@ -98,6 +98,20 @@ New modules
 Additions to existing modules
 -----------------------------
 
+* In `Algebra.Consequences.Base`:
+  ```agda
+  module Congruence (cong : Congruent₂ _≈_ _∙_) (refl : Reflexive _≈_) where
```

Comment:
This module parameterisation is currently missing equality! Evidence for the idea that my suggestion of denesting the modules would improve code readabilty!

Reply:
OK... I was clearly too influenced by my own thinking towards #2572 . Of course, for `Base`, the implicit/explicit distinction is moot, as most deployments in client modules will delegate to `Algebra.Consequences.{Setoid|Propositional}`, where the actual equality will be hidden...